SCF-Session-6/polynomial/Poly.java

```diff
@@ -0,0 +1,225 @@
+package polynomial;
+
+/**
+ * immutable class Poly which uses an array to represent single variable polynomial
+ * @author Ishita_Mathur
+ *
+ */
+public final class Poly 
```

Comment:
Why `final` is used for this class, is there any specific reason?

Reply:
To make class immutable.